docs/usage.md

```diff
@@ -510,3 +510,522 @@ This can be replicated for ROCm for AMD , by following the [instructions to set
 GaNDLF, and GaNDLF-created models, may be distributed as an [MLCube](https://mlcommons.github.io/mlcube/). This involves distributing an `mlcube.yaml` file. That file can be specified when using the [MLCube runners](https://mlcommons.github.io/mlcube/runners/). The runner will perform many aspects of configuring your container for you. Currently, only the `mlcube_docker` runner is supported. 
 
 See the [MLCube documentation](https://mlcommons.github.io/mlcube/) for more details.
+
+## Hugging Face Hub CLI
+
+This tool allows you to interact with the Hugging Face Hub directly from a terminal. For example, you can login to your account, create a repository, upload and download files, etc. It also comes with handy features to configure your machine or manage your cache.
```

Comment:
The level of detail you have provided is great, but it seems like you have pulled information verbatim from [HF site](https://huggingface.co/docs/huggingface_hub/v0.23.4/en/guides/cli). 

If the usage of `gandlf hf` is identical to `huggingface-cli`, please say so and put a link to the HF CLI usage in this section.

Reply:
> The level of detail you have provided is great, but it seems like you have pulled information verbatim from [HF site](https://huggingface.co/docs/huggingface_hub/v0.23.4/en/guides/cli).
> 
> If the usage of `gandlf hf` is identical to `huggingface-cli`, please say so and put a link to the HF CLI usage in this section.

This comment supersedes the others, so I have resolved them.